### DISCUSSION TITLE
FGFS101 Fibaro Flood Sensor %null battery

### DISCUSSION THREAD

**Discussion by majorsl** [zwave-js/zwave-js#5666] [Q&A]
Greetings.  I have two of these flood sensors and am running the latest version of zwave-js.  Both of these report %null for the battery percentage. I have tried re-pairing them and re-interviewing them multiple times and the result is always the same. I let them stay connected for awhile in hopes that when they'd check in the value would be updated (nope).

Before opening a bug report, I figured I'd ask here.  These sensors have been around for many many years and someone must be using them...

**Comment by AlCalzone** [comment]
Which application are you using?

**Reply by majorsl** [reply]
I'm seeing it right in the Z-Wave JS UI.  It occurred to me that I haven't swapped batteries in those at all.  I suppose it is possible the ones included are low and we're not getting a full interview.  I'll perhaps try that next, but I need to order some.  If it should be working then I know it's on my end and will keep troubleshooting.

<img width="1998" alt="Screenshot 2023-04-15 at 9 51 45 AM" src="https://user-images.githubusercontent.com/3105401/232228469-6c9439eb-b06c-468d-ae7a-07b7ea7e62c7.png">

**Reply by AlCalzone** [reply]
Looks like the device is reporting support for battery on 3 endpoints, but only one responds.
